src/components/Authenticate.tsx

```diff
@@ -25,6 +25,8 @@ const Authenticate = () => {
     if (stytch && !user && isInitialized) {
       const token = searchParams.get("token");
       const stytch_token_type = searchParams.get("stytch_token_type");
+      const code = searchParams.get("code");
```

Comment:
Is this code necessary? 

An OTP flow (when using the pre-built UI) doesn't involve a redirect, and thus won't hit this route. The OTP code should be entered (and auth'd) all in the login component completely by the JS SDK.

Reply:
Ah you're right, I've looked through the JS SDK and see that we can grab the user input as part of the state to pass to the authenticate call. Updated!

Reply:
To put a finer point on it, no backend authenticate call is necessary for OTP when you're using our pre-built UI components. Unlike with EML or OAuth, no redirect happens as part of the authentication flow, so the user will never hit the redirect. 

So for adding OTP to this app, all we'll need to do is the work in `Login.tsx`, no other code changes should be necessary to get OTP up and running. Mind removing the changes here to `Authenticate.tsx` and giving it a test?

Reply:
Oh woah magic! Okay just updated - appreciate the clarification.